src/adapters/azure/index.ts

```diff
@@ -7,22 +7,29 @@ import { getHandleUpload } from './handleUpload'
 import { extendWebpackConfig } from './webpack'
 
 export interface Args {
-  connectionString: string
+  connectionString?: string
   containerName: string
```

Comment:
This isn't just your PR, but we should start adding description comments on all the type properties.